docs/en/observability/triage-threshold-breaches.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[triage-threshold-breaches]]
+= Triage threshold breaches
+++++
+<titleabbrev>Threshold breaches</titleabbrev>
+++++
+
+Threshold breaches occur when an {observability} data type reaches or exceeds the threshold set in your <<custom-threshold-alert,custom threshold rule>>.
+For example, you might have a custom threshold rule that triggers an alert when the total number of log documents with a log level of `error` reaches 100.
+
+To triage issues quickly, go to the alert details page:
+
+. Go to **{observability}** → **Alerts**.
+. From the Alerts table, click the image:images/icons/boxesHorizontal.svg[More actions] icon next to the alert and select **View alert details**.
+
+The alert details page shows information about the alert, including when the alert was triggered,
+the duration of the alert, and the last status update.
+If there is a "group by" field specified in the rule, the page also includes the source.
+You can follow the links to navigate to the rule definition.
+
+Explore charts on the page to learn more about the threshold breach:
+
+[role="screenshot"]
+image::images/log-threshold-breach.png[Alert details for log threshold breach]
+
+* The page includes a chart for each condition specified in the rule.
+These charts help you understand when the breach occurred and its severity.
+* If your rule is intended to detect log threshold breaches
+(that is, it has a single condition that uses a count aggregation),
+you can run a log rate analysis, assuming you have the required license.
+Running a log rate analysis is useful for detecting significant dips or spikes in the number of logs.
+Notice that you can adjust the baseline and deviation, and then run the analysis again.
+* The page may also include an alerts history chart that shows the last 30 days.
+This chart is currently only available for rules that specify a singe condition.
+* Timelines on the page are annotated to show when the threshold was breached.
+You can hover over an alert icon to see the timestamp of the alert.
+
+Analyze these charts to better understand when the breach started, it's current
+state, and how the issue is trending.
+
+After investigating the alert, you may want to:
+
+* Click **Snooze the rule** to snooze notifications for a specific time period or indefinitely.
+* Click the image:images/icons/boxesVertical.svg[Actions] icon and select **Add to case** to add the alert to a new or existing case. To learn more, refer to <<create-cases>>.
+* Click the image:images/icons/boxesVertical.svg[Actions] icon and select **Mark as untracked**.
```

Comment:
Not sure if we have document for explaining what is `untracked` state, it would be nice to link it here. cc @shanisagiv1 @heespi

Reply:
@maryam-saeidi We do describe the untracked state in the topic about [viewing alerts](https://www.elastic.co/guide/en/observability/current/view-observability-alerts.html#filter-observability-alerts), but I felt the description didn't really help the user very much: "An alert is 'Untracked' when its corresponding rule is disabled or you mark the alert as untracked." 

We don't really describe what it means for a rule to be untracked! I think I could add a few sentences to both topics to clarify (rather than making users jump around), but I need  someone to explain what "untracked" means from a users perspective.

Reply:
Agree, that explanation does not seem to be helpful here, what would be useful for the user is an explanation of when it makes sense to untrack an alert.
It seems a separate topic from the current PR, I think it deserves its own ticket to also mention use cases that are not suitable for untrack functionality.

Reply:
Thanks for bringing this up... I think it´s fair to say that we don´t clearly explain the implications of an alert being untracked in the above docs.

@lcawl .. would you be able to help address this gap, please? CC: @shanisagiv1

Reply:
Actually looking at the [Alerting docs in Kibana](https://www.elastic.co/guide/en/kibana/master/create-and-manage-rules.html), we do a better job of explaining what "untracked" means there, so I will use a similar explanation.